packages/internal/src/files.ts

```diff
@@ -49,6 +49,7 @@ const ignoreApiFiles = [
   '**/*.test.{js,ts}',
   '**/*.scenarios.{js,ts}',
   '**/*.scenarios.{js,ts}',
```

Comment:
Is there a reason this line is duplicated?

Reply:
Just a typo I think. Nice catch